recipes/rtmidi/all/conanfile.py

```diff
@@ -50,34 +58,43 @@ def build_requirements(self):
             self.build_requires("pkgconf/1.7.4")
```

Comment:
Github won't let me suggest changing the previous line here, too, but could you change it to this to only require `pkgconf` when it's not configured directly in Conan?

```suggestion
        if self._with_alsa and not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
            self.tool_requires("pkgconf/1.9.3")
```